gpu-aware-scheduling/docs/usage.md

```diff
@@ -0,0 +1,115 @@
+# Usage with NFD and GPU-plugin
+This document explains how to get GAS working together with NFD and the GPU-plugin.
+
+To begin with, it will help a lot if you have been successfully already using the GPU-plugin with some deployments. That means your HW and cluster is most likely fine with GAS also.
+
+## GPU-plugin
+Resource management enabled version of the GPU-plugin is currently necessary for running GAS. The resource management enabled GPU-plugin version can read the necessary annotations of the PODs, and without those GPU allocations will not work correctly. It can be deployed using the device plugins operator by enabling the related CRD element, or directly via yamls. Of noteworthy is the access to the pod resource grpc server.
+
+You will need this for the plugin:
+````
+        - name: podresources
+          mountPath: /var/lib/kubelet/pod-resources
+````
+
+And therefore volumes need this:
+````
+      - name: podresources
+        hostPath:
+          path: /var/lib/kubelet/pod-resources
+````
+
+You will want in your yaml also the GPU initcontainer which installs the NFD hook. In that you will need this:
+````
+        volumeMounts:
+        - mountPath: /etc/kubernetes/node-feature-discovery/source.d/
+          name: nfd-source-hooks
+````
+
+And volumes need this:
+````
+      - name: nfd-source-hooks
+        hostPath:
+          path: /etc/kubernetes/node-feature-discovery/source.d/
+          type: DirectoryOrCreate
+````
+
+Other than those, the deployment of the GPU plugin is fairly same as without resource managing enabled. You should use a high shared-dev-number, and you need to enable the resource manager:
+````
+command: ["/usr/local/bin/intel_gpu_device_plugin", "-shared-dev-num", "300", "-v", "2", "-resource-manager", "true"]
+````
+
+There is a more relaxed RBAC-rule needed also as the plugin will need to read cluster POD information to get the resource amounts and annotations. You need to have this permission for the plugin:
+
+````
+rules:
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["list"]
+````
+
+The GPU plugin initcontainer needs to be used in order to get the extended resources created with NFD. The initcontainer installs the required NFD-hook into the host system.
+
+## NFD
+The latest upstream NFD will be sufficient. You can use it to publish the GPU extended resources and GPU-related labels printed by the hook installed by the GPU-plugin initcontainer.
```

Comment:
Maybe link some tag or commit here for comparison?

Reply:
I need to improve this a lot here